src/containers/ErrorMessages.js

```diff
@@ -4,6 +4,39 @@ import { ToastNotificationList, ToastNotification } from 'patternfly-react';
 import { withRouter } from 'react-router-dom';
 import { dismiss, dismissAll } from '../actions/errors';
 
+const MOBILECLIENT = 'mobileclients';
+const ALREADYEXISTS = 'AlreadyExists';
+
+function errorMessage(error) {
+  try {
+    switch (error.response.data.details.kind) {
+      case MOBILECLIENT:
+        // case to handle MobileClient CRs
+        return { dispalyMessage: mobileClientError(error), message: error.message };
+      default:
+        return { message: error.message };
+    }
+  } catch (TypeError) {
+    // catch any error if error.response.data.details.kind does not exist
+    return { message: error.message };
+  }
+}
+
+function mobileClientError(error) {
+  try {
+    switch (error.response.data.reason) {
+      case ALREADYEXISTS:
+        // handle if mobileClient exists
+        return 'App name currently exists';
```

Comment:
Do you have the app name here? If you have, I think that an error like `An app named ${app.name} already exists`.
If we don't have the app name here, I believe the message would be better as 'An app with the same name already exists`